docs/changelog.md

```diff
@@ -289,7 +289,7 @@ Changelog for commit "[fix(Core/Player): Use SkillLineAbility.dbc to determine p
 -    //uint32    racemaskNot;                                // 5        m_excludeRace
 -    //uint32    classmaskNot;                               // 6        m_excludeClass
 -    uint32    req_skill_value;                              // 7        m_minSkillLineRank
--    uint32    forward_spellid;                              // 8        m_supercededBySpell
+-    uint32    forward_spellid;                              // 8        m_supersededBySpell
```

Comment:
why you removed the comment m_* ?

Reply:
I did not remove the comment. Spelling was corrected to m_supersededBySpell from m_supercededBySpell

Reply:
It wasnt removed, the C was changed to S. But that is wrong, as in the core it is defined with C

Reply:
While these changes are appreciated, the variables and stuff needs to be checked with how they are defined in the core before changes are made.